### PR TITLE
Handle case where sendto fails on interface that does not support it.

### PR DESCRIPTION
**Pull request by tom24**
This can be because the interface is a VPN tunnel or is DOWN.

### CODE REVIEW

File: wsdiscovery/threaded.py [hunk 1]
```diff
@@ -210,7 +210,14 @@ def _sendMsg(self, msg):
                 self._seqnum += 1
         else:
             for sock in list(self._multiOutUniInSockets.values()):
-                sock.sendto(data, (msg.getAddr(), msg.getPort()))
+                try:
+                    sock.sendto(data, (msg.getAddr(), msg.getPort()))
+                except OSError as e:
+                    # sendto will fail for interfaces that do not support multicast or are not up.
+                    # An example of the first case is a wireguard vpn interface.
+                    # In either just log as debug and ignore the error.
```
Comment (petri) [inline wsdiscovery/threaded.py:218]: I guess this is supposed to say "In either case" (just log...)

**Review by petri — CHANGES_REQUESTED** [review]
Seems good to me. There's just a small typo I guess in docstr.

**Comment by tom24** [comment]
I corrected the typo.

**Comment by andreikop** [comment]
Thank you!

**Comment by yateya** [comment]
Thanks for fixing this bug. Will it be released to [pypi repo](https://pypi.org/project/WSDiscovery) soon?

**Comment by andreikop** [comment]
@yateya , well, not soon, but it is released now.

**Pull request merged**